developer-experience/recipes/onboarding-guide-template.md

```diff
@@ -0,0 +1,35 @@
+# Onboarding Guide Template
+
+When developing an onboarding document for a team, it should contain details of engagement scope, team processes, codebase, coding standards, team agreements, software requirements and setup details.
+
+## Overview and Goals
+
+* List a few sentences explaining the high-level summary and the scope of the engagement.
+* Consider adding any additional background and context as needed.
+* Include the goals of this engagement and what the team is trying to achieve.
+
+## Contacts
+
+* List a few of the main contacts from the engagement both from the customer and Microsoft team.
+* Consider including the roles of these main contacts so that the team knows who to reach out to depending on the situation.
+
+## Team Agreement and Code of Conduct
+
+* Include the team's code of conduct or agreement that defines a set of expectation from each team member and how the team has agreed to operate.
```

Comment:
Might suggest leveraging links where possible within this section. If they want to repeat things its probably aspects of each the team feels is especially critical (for example working hours might be super critical when working across timezones).